requirements.txt

```diff
@@ -0,0 +1,6 @@
+Cython==0.25.2
+confluent-kafka==0.11.0
```

Comment:
This one is redundant by row 4, which is also not required since it's installed by eventsourcing_helpers

Reply:
I see no issue in being explicit :-D

Reply:
OK I can't argue with that. But please install only one of the confluent-kafka libraries at least, my vote goes to this one :)

Reply:
Haha, ok ;)